releasenotes/notes/wrap-method-311-147d254d4b40e805.yaml

```
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixed bad handling of ``__init_subclass__`` by `~qiskit.utils.wrap_method`
```

If we want the releasenotes to link this correctly I think we need to mark this as a function
```suggestion
    Fixed bad handling of ``__init_subclass__`` by :func:`~qiskit.utils.wrap_method`
```